apps/server/src/external/demo/app.js

```diff
@@ -0,0 +1,69 @@
+/*eslint-env browser*/
+/**
+ * This is a very minimal example for a websocket client
+ * You could use this as a starting point to creating your own interfaces
+ *
+ * This example does not handle disconnections
```

Comment:
I think it does now, does it not?

Reply:
haha yes it dose

Reply:
feel free to add clarifying comments to guide the user if you have some

Reply:
I think it is good enough as it is, we can create a help page for this outside the code base and add a link here
Good work